src/cmd/completion.go

```diff
@@ -62,7 +62,7 @@ func completionCommands(cmd *cobra.Command, _ []string, _ string) ([]string, cob
 	commandNames := []string{}
 	commands := cmd.Root().Commands()
 	for _, command := range commands {
-		if strings.Contains(command.Name(), "complet") {
+		if strings.Contains(command.Name(), "complete") {
```

Comment:
I wonder if there was a deliberate desire to search for a sub-string or check the prefix here? It's curious that it uses `strings.Contains` instead of a simpler comparison.

It will be better to leave this out until we are really sure.

Reply:
That was me trying to be smart. Possibly I was trying to catch multiple commands with a single function call. Its weird and should not probably exist in the first place.

Reply:
This change is still present in the new version of the branch.  It shouldn't be part of the commit to fix existing spelling mistakes.

If we are sure that the *complet* should be changed to *complete* then it should be done in a separate commit with an explanation about why it's safe to do so, and replacing `strings.Contains` with a simpler comparison.

Reply:
I took the liberty to drop this in https://github.com/containers/toolbox/pull/1166